namesgenerator.js

```diff
@@ -93,6 +93,6 @@ function randnum(n) {
 }
```

Comment:
Sorry for the delay in responding, and thanks for pointing out the bug. I would prefer that you modify line 92 and change `(max - min + 1)` to `(max - min)` instead of changing line 96. Thanks!